feat: More small fixes to the SDK Doctor

rafaeelaudibert opened the pull request:
- Run Dagster job in half the time, seems to be doing well Clickhouse-wise. If still too slow we'll need to batch-load data from Clickhouse for more than one team by falling back to raw Clickhouse SQL. This has some risk so I'd like to avoid that for now.
- Parse query only once to speed up the Dagster job
- Use `created_at` to report the timestamp, avoid dates in the future
- Add link to version to open SQL editor (cross-selling babyyy)
- Add `sortableSemVer` as a HogQL query! This will let people use this anywhere, it's a pretty cool function, will add it to the docs soon
- Trigger some events to track SDK Doctor usage

<img width="555" height="121" alt="image" src="https://github.com/user-attachments/assets/b62f6642-260d-4229-9a0d-68e111b731b6" />

rafaeelaudibert committed:
refactor: Run this more in parallel

This will make it run faster, it's currently taking almost 3 hours

rafaeelaudibert committed:
refactor: Parse query only once

No need to recreate the query every time, just have it preprocessed once

rafaeelaudibert committed:
refactor: Use `created_at`

This more correctly stores when an event was processed

rafaeelaudibert committed:
refactor: Add link to open events in SQL editor

rafaeelaudibert committed:
feat: Add `sortableSemver` HogQL function

This function properly sorts Semver items in a reasonable order rather than simple alphabetical order, i.e. it properly ranks 1.10.0 above both 1.1.0 and 1.2.0 (and 1.9.213323232 even).

This is used in the SDK Doctor only for now, but we could expand to use it everywhere.

Also moved some functions around to a better place

rafaeelaudibert committed:
feature: Capture events when actions are taken in the SDK Doctor sidebar

rafaeelaudibert committed:
fix: Filter by `created_at` rather than `timestamp`

Make it consistent Rafa, please

github-actions[bot] committed:
Update UI snapshots for `chromium` (1)

github-actions[bot] committed:
Update query snapshots

rafaeelaudibert committed:
fix: Handle survey functions in the right place

Gilbert09 reviewed (approved):
If you could just move the implementation into `posthog/hogql/functions/posthog.py` pleeeease 🙏

rafaeelaudibert committed:
fix: Stop considering no team as error

rafaeelaudibert committed:
feat: Add tags context

rafaeelaudibert committed:
refactor: Revert to timestamp

There are no indexes on `created_at`, so let's go back for now

rafaeelaudibert committed:
fix: Use right product

rafaeelaudibert committed:
refactor: Extract `sortableSemVer` to the `posthog.py` file

We don't need to include that on `printer.py`, we can solve it inline

rafaeelaudibert committed:
fix: Use proper variable

rafaeelaudibert committed:
fix: Use `node.args` for survey functions

rafaeelaudibert committed:
refactor: Move function back

HOGQL_POSTHOG_FUNCTIONS are handled differently and they don't support suffix args

rafaeelaudibert committed:
Merge branch 'master' into more-small-tweaks-sdk-doctor

rafaeelaudibert committed:
refactor: Make frontend query faster

github-actions[bot] committed:
Update query snapshots
